File: Client/Udomitelj.js
export class Udomitelj
{
    constructor(ime,prezime,grad,ulica,postanskibroj,pas)
    {
        this.ime=ime,
        this.prezime=prezime,
        this.grad=grad,
        this.ulica=ulica,
        this.postanskibroj=postanskibroj,
        this.pas=pas
    }

    crtaj(host)
    {
        var tr=document.createElement("tr");
        host.appendChild(tr);

        var u=document.createElement("td");
        u.innerHTML=this.ime;
        tr.appendChild(u);

        var u=document.createElement("td");
        u.innerHTML=this.prezime;
        tr.appendChild(u);

        var u=document.createElement("td");
        u.innerHTML=this.grad;
        tr.appendChild(u);

        var u=document.createElement("td");
        u.innerHTML=this.ulica;
        tr.appendChild(u);

        var u=document.createElement("td");
        u.innerHTML=this.postanskibroj;
        tr.appendChild(u);

        var u=document.createElement("td");
        u.innerHTML=this.pas.ime;
        tr.appendChild(u);
    }
}

File: Models/Udomitelj.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models{
public class Udomitelj
    {
        [Key]
        public int ID { get; set; }

        [MaxLength(50)]
        public string Ime { get; set; }
         [MaxLength(50)]

        public string Prezime { get; set; }
        public string Grad { get; set; }
        public string Ulica { get; set; }

        
        public int PostanskiBroj { get; set; }
        
        //[JsonIgnore]
        public Azil azil{get; set;}

    //[JsonIgnore]
        public Pas pas {get;set;}

    }}